### PR TITLE
add towncrier

### PR DESCRIPTION
**Pull request by aviramha**

### CODE REVIEW

File: changelog.d/+towncrier.changed.md [hunk 1]
```diff
@@ -0,0 +1 @@
+CHANGELOG - changed to use towncrier
```
Comment (t4lz) [inline changelog.d/+towncrier.changed.md:1]: So meta

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,20 +1,12 @@
...
 
-Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+This project uses [*towncrier*](https://towncrier.readthedocs.io/) and the changes for the upcoming release can be found in <https://github.com/metalbear-co/mirrord/tree/main/changelog.d/>.
 
-### Fixed
-
-- Possible bug when bound address is bypassed and socket stays in `SOCKETS` map.
```
Comment (t4lz) [inline CHANGELOG.md:12]: Isn't this entry still missing?

Comment (aviramha) [reply CHANGELOG.md:12]: good catch, forgot to stage it.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,20 +1,29 @@
...
-Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+This project uses [*towncrier*](https://towncrier.readthedocs.io/) and the changes for the upcoming release can be found in <https://github.com/metalbear-co/mirrord/tree/main/changelog.d/>.
 
-### Fixed
+<!-- towncrier release notes start -->
+
+## [3.32.0](https://github.com/metalbear-co/mirrord/tree/3.32.0) - 2023-03-07
```
Comment (t4lz) [inline CHANGELOG.md:11]: Is this on purpose? Shouldn't this only happen on release? Can we maybe have it automatically happen in release.yaml?

**Review by t4lz — APPROVED** [review]

**Pull request merged**